.travis.yml

```diff
@@ -9,7 +9,7 @@ install:
 script:
   - git remote set-url origin https://${GITHUB_TOKEN}@github.com/bluerobotics/ping-viewer
   - mkdocs build
-  - htmlproofer --empty-alt-ignore --file-ignore "./site/404.html" --url-ignore "https://fonts.gstatic.com" || travis_terminate 1
+  - htmlproofer --empty-alt-ignore --file-ignore "./site/404.html" --url-ignore "https://fonts.gstatic.com" site || travis_terminate 1
```

Comment:
What exactly does this do? Is it a functional change?

Reply:
this runs the htmlproofer in what is effectively `/` for the website. Without it, absolute links are broken. Instead of looking for images in `./site/images`(where they are), it tries to find them in `./images` and it fails.